index.js

```diff
@@ -1,54 +1,44 @@
 const exec = require('child_process').exec
 const createDebugger = require('debug')
 const path = require('path')
+const _ = require('underscore')
 
 const depsCommand = `npm ls --parseable`
 
-function splitOnNewLine(string) {
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
+function mapOutputToDepList(string) {
   return string.split('\n')
 }
 
 function pathOfPackage(string) {
-  // pass dependency chain with
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
   return path.join(string, "package.json")
 }
 
 function packageFor(string) {
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
   return require(string)
 }
 
-function keyFromPackage(key) {
+function keysFromPackage(keys) {
   return object => {
-    let dbgr = createDebugger(arguments.callee.name)
-    //dbgr(JSON.stringify(object))
-
-    return { packageName: object.name, version: object.version, keyValue: object[key] }
+    return _.pick(object, keys)
   }
 }
 
 function tailOfPath(string) {
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
   return string.split('/').pop()
 }
 
 function removeEmptyStrings(item) {
   return item !== ''
 }
 
-function start(key, cb) {
-  let dbgr = createDebugger(arguments.callee.name)
-  dbgr("In start function")
+Array.prototype.removeThisPackage = function() {
```

Comment:
Cool I did not notice rest/tail in the underscore docs. Thats what I really want and `.tail()` would read well were I thought `.slice(1)` was not as revealing.